SU-SWS/ace-sdssgryphon
# SDSS-84: updates to the style options

# READY FOR REVIEW

# Summary
- Updates to style list option in WYSIWYG editor.
- Updated to `rem` from `px`

# Review By (Date)
- 3/10

# Urgency
- Normal

# Steps to Test

1. Checkout this change
2. Update and clear caches.
3. Edit a text area paragraph
4. Notice the list of styles has been edited and renamed.
5. Use the /qa-typography-styles for examples.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SDSS-84](https://stanfordits.atlassian.net/browse/SDSS-84)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
